static/sass/_pattern_footer.scss

```diff
@@ -42,4 +42,9 @@
       right: -0.8rem;
     }
   }
+  @media (max-width: $breakpoint-x-small) {
+    .p-footer {
+      clip-path: polygon(100% 0%, 124% 134%, 0% 124%, 0% calc(0% + 4rem));
```

Comment:
Why adjust this clip-path? You have increased the width of the sides just hasn't made a difference to the final result. Did you mean to decrease the height of the diagonal at the top?

Reply:
@anthonydillon ooh, I forgot to actually adjust it 😅
> Did you mean to decrease the height of the diagonal at the top?

yeah it should be `clip-path: polygon(100% 0%, 124% 134%, 0% 124%, 0% calc(0% + 4rem));`
![Screenshot from 2021-11-17 01-45-25](https://user-images.githubusercontent.com/44097148/142058952-de600068-973b-420e-a11c-ecd765bd8e91.png)